examples/postgres/main.go

```diff
@@ -0,0 +1,5 @@
+package main
+
+func main() {
+
+}
```

Comment:
If we are not allowing user to run it, we might as well just leave out this file